configuration/configuration.go

```diff
@@ -72,23 +72,23 @@ const (
 	// Public Client ID for logging into Auth service via OAuth2
 	varPublicOauthClientID = "public.oauth.client.id"
 
-	// Keycloak
-	varKeycloakSecret           = "keycloak.secret"
-	varKeycloakClientID         = "keycloak.client.id"
-	varKeycloakDomainPrefix     = "keycloak.domain.prefix"
-	varKeycloakRealm            = "keycloak.realm"
-	varKeycloakTesUserName      = "keycloak.testuser.name"
-	varKeycloakTesUserSecret    = "keycloak.testuser.secret"
-	varKeycloakTesUser2Name     = "keycloak.testuser2.name"
-	varKeycloakTesUser2Secret   = "keycloak.testuser2.secret"
-	varKeycloakURL              = "keycloak.url"
-	varKeycloakEndpointAdmin    = "keycloak.endpoint.admin"
-	varKeycloakEndpointAuth     = "keycloak.endpoint.auth"
-	varKeycloakEndpointToken    = "keycloak.endpoint.token"
-	varKeycloakEndpointUserinfo = "keycloak.endpoint.userinfo"
-	varKeycloakEndpointBroker   = "keycloak.endpoint.broker"
-	varKeycloakEndpointAccount  = "keycloak.endpoint.account"
-	varKeycloakEndpointLogout   = "keycloak.endpoint.logout"
+	// OAuthService
+	varOAuthServiceSecret           = "oauth.service.secret"
+	varOAuthServiceClientID         = "oauth.service.client.id"
+	varOAuthServiceDomainPrefix     = "oauth.service.domain.prefix"
+	varOAuthServiceRealm            = "oauth.service.realm"
+	varOAuthServiceTesUserName      = "oauth.service.testuser.name"
+	varOAuthServiceTesUserSecret    = "oauth.service.testuser.secret"
+	varOAuthServiceTesUser2Name     = "oauth.service.testuser2.name"
+	varOAuthServiceTesUser2Secret   = "oauth.service.testuser2.secret"
+	varOAuthServiceURL              = "oauth.service.url"
+	varOAuthServiceEndpointAdmin    = "oauth.service.endpoint.admin"
+	varOAuthServiceEndpointAuth     = "oauth.service.endpoint.auth"
+	varOAuthServiceEndpointToken    = "oauth.service.endpoint.token"
+	varOAuthServiceEndpointUserinfo = "oauth.service.endpoint.userinfo"
+	varOAuthServiceEndpointBroker   = "oauth.service.endpoint.broker"
+	varOAuthServiceEndpointAccount  = "oauth.service.endpoint.account"
+	varOAuthServiceEndpointLogout   = "oauth.service.endpoint.logout"
```

Comment:
Fine to change the variable names, not the values yet, please.

Reply:
The migration path should be : 
- add new env variables in our cluster, while the old ones still exist.
- remove refs to old env variables in our code.